app/src/main/java/org/simple/clinic/bloodsugar/entry/BloodSugarEntryUi.kt

```diff
@@ -18,5 +18,8 @@ interface BloodSugarEntryUi {
   fun hideDateErrorMessage()
   fun setDateOnInputFields(dayOfMonth: String, month: String, twoDigitYear: String)
   fun showDateOnDateButton(date: LocalDate)
+  fun showEditRadomBloodSugarTitle()
+  fun showEditPostPrandialBloodSugarTitle()
+  fun showEditFastingBloodSugarTitle()
```

Comment:
Can we rename the corresponding methods for setting the title when entering a new blood sugar reading as well?

Example:
`showRandomBloodSugarTitle` -> `showEnterRandomBloodSugarTitle`.

Reply:
In later commits, I am planning on replacing these methods with a single method with just the measurement type parameter. Should I still replace these methods names in this PR?

Reply:
If you're going to do this, then it's fine. Let's just take a look at it in the end after the feature is complete.